src/graphql/mongoose_graphql.erl

```diff
@@ -19,8 +19,16 @@
                      vars := map(),
                      authorized := boolean(),
                      ctx := map()}.
+-type context() :: map().
+-type object() :: term().
+-type field() :: binary().
+-type args() :: term().
```

Comment:
Is it possible for args to be not a `map()` type?

Reply:
ok, probably no.